portal/templates/cluster_node_details.html

```diff
@@ -0,0 +1,90 @@
+{%extends "base.html"%}
+
+{%block title%}
+  Node Details
+{%endblock%}
+
+{%block body%}
+<section id="cluster-profile" class="bg-light">
+  <div class="container col-lg-12 mx-auto">
+
+    <div class="row">
+      <div class="col-lg-12 mx-auto">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item"><a href="{{url_for('dashboard')}}">Home</a></li>
+          <li class="breadcrumb-item"><a href="{{url_for('list_clusters')}}">Clusters</a></li>
+          <li class="breadcrumb-item"><a :href="'/clusters/'+cluster_name">{[{cluster_name}]}</a></li>
+          <li class="breadcrumb-item active">{[{node_name}]}</li>
+        </ol>
+        <h1>Node: {[{node_name}]}</h1>
+        <hr/>
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="card h-100">
+              <div class="card-header">
+                Monitoring status
+              </div>
+              <div class="card-body">
+                <div class="row">
+                  <div class="col-sm-12 col-md-6 col-lg-6">
+                    <!-- CPU -->
+                    <iframe :src="'https://monitoring.umich-prod.slateci.net/d-solo/6u_Akx7Gk/cluster-monitoring-startout-copy?orgId=1&theme=light&var-DS_PROMETHEUS=SLATE&var-cluster='+cluster_name+'&var-name='+node_name+'&panelId=310'" width="100%" height="330" frameborder="0" style="pointer-events: none"></iframe>
```

Comment:
Same situation here. By using the `:src="url-string"`, you are indicating that this is going to use VueJS variables. The `cluster_name` and `node_name` that you are trying to call here was not assigned in the VueJS data() return function.